spec/controllers/cases_controller_spec.rb

```diff
@@ -56,7 +56,7 @@
 
     it 'decorates the collection of case transitions' do
       expect(assigns(:case_transitions)).to be_an_instance_of(Draper::CollectionDecorator)
-      expect(assigns(:case_transitions).map(&:class)).to eq [ CaseTransitionDecorator ]
+      expect(assigns(:case_transitions).map(&:class)).to eq [ CaseTransitionDecorator, CaseTransitionDecorator ]
```

Comment:
This is making my spidey-senses tingle - something not quite right here?

Reply:
Gotcha! Yeah, maybe there is a way to express the intent of the test a little more explicitly, then, but your explanation helped a lot! - probably looks better in the file itself than out of context here